### PR TITLE
fixed the words order

### PR DESCRIPTION
**Pull request by niaomingjian**

### CODE REVIEW

**Comment by niaomingjian** [comment]
I signed it

**Comment by tengqm** [comment]
/hold
This doesn't look like a valid fix.

**Review by Aut0R3V — APPROVED** [review]

**Comment by Aut0R3V** [comment]
Maybe the documentation makes sense. It'll  be great if you add some more details in the description.

**Comment by irvifa** [comment]
Hi @sftim would you mind taking a look on this if it’s valid or not? I need another opinion

File: content/en/docs/concepts/extend-kubernetes/operator.md
Comment (sftim) [inline content/en/docs/concepts/extend-kubernetes/operator.md:63]: Imagine that this is short for: “At the core of the Operator is code. That code is there to tell the API server how to make&hellip;”

The original text makes more sense @niaomingjian so I could not accept this as-is. If you'd like to propose a different working, please feel free. It will help reviewers if you summarize the intent of that change in the PR description.

**Review by sftim — CHANGES_REQUESTED** [review]

File: content/en/docs/concepts/extend-kubernetes/operator.md
Comment (sftim) [reply content/en/docs/concepts/extend-kubernetes/operator.md:63]: (By the way, operators can and often do incorporate more than one control loop)

**Comment by kbhawkey** [comment]
👋 @niaomingjian .
Just checking if you are working on this PR?

**Pull request closed without merging**
